tests/BenchmarkDotNet.Tests/Attributes/VerifiedFiles/ParamsAllValuesVerifyTests.BenchmarkShouldProduceSummary_WithAllValuesOfBool.verified.txt

```diff
@@ -7,9 +7,9 @@ Frequency: 2531248 Hz, Resolution: 395.0620 ns, Timer: TSC
   DefaultJob : extra output line
 
 
-    Method | ParamProperty |     Mean |   Error |  StdDev |
----------- |-------------- |---------:|--------:|--------:|
- Benchmark |         False | 102.0 ns | 6.09 ns | 1.58 ns | ^
- Benchmark |          True | 202.0 ns | 6.09 ns | 1.58 ns | ^
+Method    | ParamProperty |     Mean |   Error |  StdDev |
+--------- |-------------- |---------:|--------:|--------:|
+Benchmark | False         | 102.0 ns | 6.09 ns | 1.58 ns | ^
+Benchmark | True          | 202.0 ns | 6.09 ns | 1.58 ns | ^
```

Comment:
Can you re-add the space in front of the first column?

Reply:
The removed space was due to adding spaces to start of columns which was the behaviour for all the columns. You would see more spaces if texts of two rows had different length. Now we add spaces to justify based on the given style. For a text column with default styling value (left justification) we only add spaces to the end of column

Reply:
I understand, but I would like to avoid other changes to the table besides justification.

Reply:
> I understand, but I would like to avoid other changes to the table besides justification.

Ok, This is fixed in my latest commit.